android/src/app/organicmaps/bookmarks/Holders.java

```diff
@@ -275,6 +276,19 @@ void setSize(@PluralsRes int phrase, int size)
       mSize.setText(mSize.getResources().getQuantityString(phrase, size, size));
     }
 
+    void setZeroSize()
+    {
+      setSize(R.plurals.objects, 0);
+    }
+
+    void setDetailedSize(int tracks, int bookmarks)
+    {
+      Resources resources = mSize.getResources();
```

Comment:
Use `final` if you don't modify the variable.
```suggestion
      final Resources resources = mSize.getResources();
```